packages/core/contracts/polygon/OracleChildTunnel.sol

```diff
@@ -96,9 +97,37 @@ contract OracleChildTunnel is OracleBaseTunnel, OracleAncillaryInterface, FxBase
         uint256 time,
         bytes memory ancillaryData
     ) public view override onlyRegisteredContract() returns (int256) {
-        bytes32 priceRequestId = _encodePriceRequest(identifier, time, ancillaryData);
+        bytes32 priceRequestId = _encodePriceRequest(identifier, time, _stampAncillaryData(ancillaryData, msg.sender));
         Price storage lookup = prices[priceRequestId];
         require(lookup.state == RequestState.Resolved, "Price has not been resolved");
         return lookup.price;
     }
+
+    /**
+     * @dev We don't handle specifically the case where `ancillaryData` is not already readily translateable in utf8.
+     * For those cases, we assume that the client will be able to strip out the utf8-translateable part of the
+     * ancillary data that this contract stamps.
+     */
+    function _stampAncillaryData(bytes memory ancillaryData, address requester) internal view returns (bytes memory) {
```

Comment:
@chrismaree @mrice32 this the ancillary data I'm thinking of stamping. I don't think the child tunnel address or root tunnel address is neccessary to stamp since (1) the root tunnel will be the one requesting the price on L1 and (2) the child tunnel is linked with the root tunnel.
Thoughts?